charts/loki-simple-scalable/README.md

```diff
@@ -219,7 +219,7 @@ For exmaple, to use MinIO (deployed separately) as your backend, provide the fol
 
 ```yaml
 loki:
-  config:
+  config: |
```

Comment:
@Alex3k these changes need to be made to the gotmpl file instead of the actual README. CI missed this because we we're currently ignoring generating helm-docs for this chart, but as soon as we turn it back on this change will get lost.